src/Simulation.jl

```diff
@@ -283,7 +283,7 @@ function rops(ssys::SystemSimulation, t)
     for inter in ssys.interfaces
         if inter isa FragmentBasedReactiveFilmGrowthInterfaceConstantT
             kfs, krevs = getkfskrevs(inter)
-            rops!(ropmat, inter.rxnarray, inter.fragmentbasedrxnarray, cstot, kfs, krevs, vV[inter.domaininds[1]], start)
```

Comment:
Does this change rops! syntax for other simulations?

Reply:
No it doesn't, it's a dispatch of `rops!`